docs/user-guide/rules.md

```diff
@@ -156,6 +156,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`selector-class-pattern`](../../src/rules/selector-class-pattern/README.md): Specify a pattern for class selectors.
 - [`selector-combinator-space-after`](../../src/rules/selector-combinator-space-after/README.md): Require a single space or disallow whitespace after the combinators of selectors.
 - [`selector-combinator-space-before`](../../src/rules/selector-combinator-space-before/README.md): Require a single space or disallow whitespace before the combinators of selectors.
+- [`selector-descendant-combinator-no-non-space`](../../src/rules/selector-descendant-combinator-no-non-space/README.md): Disable non-spaces characters of the descendant combinator in descendant selectors.
```

Comment:
I think this would be clearer as "Disable non-space characters for the descendant combinator of a selector."